pkg/resource/deploy/source_eval.go

```diff
@@ -198,11 +200,16 @@ func (iter *evalSourceIterator) forkRun(opts Options) {
 				DryRun:         iter.src.dryRun,
 				Parallel:       opts.Parallel,
 			})
+
+			if err == nil && progerr == "A97455BA-8A80-42A5-8639-53CD49E88D75" {
+				return result.Bail()
+			}
```

Comment:
Note: i'm only doing this now because i don't know how to actually regen protobufs.  This should actually just be a bool in the value that langhost.Run returns back.  i'll work with another dev to be able to do that.